### PR TITLE
samples: Add nRF9161 Wi-Fi tests to twister

### PR DESCRIPTION
**Pull request by krish2718**
Apart from checking the build these are also needed for generating memory footprints.

### CODE REVIEW

**Comment by divipillai** [comment]
The docs for these samples need updates. I will be working on the docs.

**Review by sachinthegreen — APPROVED** [review]

File: samples/cellular/location/sample.yaml [hunk 1]
```diff
@@ -34,7 +34,8 @@ tests:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns
+      - nrf9161dk_nrf9161_ns
```
Comment (trantanen) [inline samples/cellular/location/sample.yaml:37]: We'd need to add nrf9161dk_nrf9161_ns to all configurations of this file. Should I do that with a separate PR?

Comment (krish2718) [reply samples/cellular/location/sample.yaml:37]: Yes please that would be better, this PR is only for Wi-Fi parts.

Comment (divipillai) [reply samples/cellular/location/sample.yaml:37]: I plan to cover the document updates for the location sample as part of this PR. Should that be covered in the new PR?

Comment (krish2718) [reply samples/cellular/location/sample.yaml:37]: Preferably, as I would then need to add you as a colloborator.

Comment (trantanen) [reply samples/cellular/location/sample.yaml:37]: PR https://github.com/nrfconnect/sdk-nrf/pull/13048 for Location sample

Comment (trantanen) [reply samples/cellular/location/sample.yaml:37]: You can remove the changes in this file from this PR.

**Review by trantanen — APPROVED** [review]

**Comment by divipillai** [comment]
I have covered the doc updates of shell sample here: https://github.com/nrfconnect/sdk-nrf/pull/13052/files#diff-7fd937b453cbda230e07e8f3fd883f26ba21b2b9af7631819d70c4f8a997dd47 . So, we are good from doc side.
